# This is a message to hold data for pF/LMPC controller
Header header
float64[] z_x 
float64[] z_y
float64[] z_vx
float64[] z_s
float64[] SS_x
float64[] SS_y
float64[] SS_vx
float64[] SS_s
float64[] z_fore_x
float64[] z_fore_y
float64[] z_iden_x
float64[] z_iden_y